python/cudf/cudf/core/series.py

```diff
@@ -5008,8 +5008,73 @@ def _get_dt_field(self, field):
             data=out_column, index=self.series._index, name=self.series.name
         )
 
-    def ceil(self, field):
-        out_column = self.series._column.ceil(field)
+    def ceil(self, freq):
+        """
+        Perform ceil operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        freq : str
+            One of ["D", "H", "T", "S", "L", "U", "N"]
+            Must be a fixed frequency like ‘S’ (second) not ‘ME’ (month end).
```

Comment:
Extinguishing the smart quotes. 👨‍🚒🧯🔥 
```suggestion
            Must be a fixed frequency like 'S' (second) not 'ME' (month end).
```
This probably happened via copy-paste from the rendered docs. The [source](https://github.com/pandas-dev/pandas/blob/cb839779749a2cd71cbb8968282fff55f430ad6b/pandas/core/arrays/datetimelike.py#L1603-L1604) doesn't use smart quotes, but they get converted to smart quotes when rendered in [the Sphinx docs](https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.ceil.html#pandas.Series.dt.ceil).

Ditto below.